backend/game/cards/discard_cards/multi_manipulation_cards/test_vast_storm.py

```diff
@@ -0,0 +1,16 @@
+from game.cards.discard_cards.multi_manipulation_cards.vast_storm import VastStorm
+
+
+def test_vast_storm_adds_2_victory_points(player):
+    VastStorm().immediate_effect(player, None)
+    assert player.victory_points == 2
+
+
+def test_vast_storm_subtracts_3_health(player):
+    player.current_health = 8
+    VastStorm().immediate_effect(player, None)
+    assert player.current_health == 7
```

Comment:
This math doesn't add up to the test name

Reply:
vast storm removes 1 health point, set initial to 8 then trigger method which removes 1 health point outcome is 7